composer.json

```
@@ -22,7 +22,7 @@
         "easy-doc/easy-doc": "0.0.0 || ^1.2.3",
         "friendsofphp/php-cs-fixer": "^3.57",
         "gregwar/rst": "^1.0.6",
-        "phpstan/phpstan": "~2.1.30",
+        "phpstan/phpstan": "2.1.31",
```

```suggestion
        "phpstan/phpstan": "~2.1.31",
```
Unless we have issues with upper versions?

---

We do, 2.1.32 fails on 8.1 - 8.3, but passe for 8.4. I'd prefer to solve it separately.